_pytest/junitxml.py

```diff
@@ -53,6 +53,13 @@ def repl(matchobj):
             return unicode('#x%04X') % i
     return py.xml.raw(illegal_xml_re.sub(repl, py.xml.escape(arg)))
 
+def record_property(name, value):
```

Comment:
I understand that you wanted to mimic GTest's, but perhaps this wold be better named `record_xml_property`?